src/firebase-dynamic-links.ts

```diff
@@ -11,7 +11,7 @@ export class FirebaseDynamicLinks {
    * Take note of your project `Web Api Key` from [setting page](https://console.firebase.google.com/project/_/settings/general/) of the Firebase console.
    */
   constructor(webApiKey: string) {
-    if (webApiKey == null) {
+    if (webApiKey === null || webApiKey === undefined) {
```

Comment:
@zerefdev having the `==` opeartor checks both `null` & `undefined`. this change is redundant.

Reply:
Correct. The only reason I changed that line is because my linter was yelling at me, "eqeqeq" rule => `==` checks for "null" as string as well. It's unlikely that someone will provide "null" as key but just to be safe.